routing/routing_integration_tests/route_test.cpp

```diff
@@ -559,4 +559,14 @@ namespace
 
     TEST_EQUAL(route.second, RouterResultCode::NoError, ());
   }
+
+  UNIT_TEST(NoCrash_HelsinkiPiter)
```

Comment:
nit: s/NoCrash/AreMwmsNear/

all tests are supposed to be "no crash"

Reply:
fixed